llvm/include/llvm/CodeGen/MachineInstr.h

```diff
@@ -1818,6 +1818,9 @@ class MachineInstr
   /// preferred.
   void addOperand(const MachineOperand &Op);
 
+  /// Inserts Ops BEFORE It. Can untie/retie tied operands.
+  void insert(mop_iterator It, ArrayRef<MachineOperand> Ops);
```

Comment:
Another compiler I worked on would always call these sort of parameters `InsertBefore` or `InsertAfter` (instead of `It` here) to make the semantics clear. Maybe that would be neat here to show the semantics? On the other hand I haven't seen this pattern anywhere else in LLVM so no strong opinion.

Reply:
yeah, I was thinking about that this weekend; would be a nice change. Will do.

Reply:
done in https://github.com/llvm/llvm-project/pull/67699/commits/6ba0e06260532ca0cd0e2b54b0f8968951d2be23